packages/client/src/components/activityIndicator/activityIndicator.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+import styles from './activityIndicator.module.css'
+
+export const ActivityIndicator: React.FC = () => {
+  return <div className={styles.activityIndicator}></div>
+}
```

Comment:
Тут можно сократить, использовать краткую форму для метода + поправить закрывающий тэг, раз у него нет детей

```suggestion
export const ActivityIndicator: React.FC = () => (<div className={styles.activityIndicator}/>)
```

Reply:
Сократил.